src/main/java/ladder/view/ResultView.java

```diff
@@ -1,52 +1,70 @@
 package ladder.view;
 
 import ladder.game.LadderGame;
+import ladder.game.Participants;
+import ladder.game.Prizes;
+import ladder.structure.Ladder;
+import ladder.structure.LineOfLadder;
 
 import java.util.Collections;
 import java.util.List;
+import java.util.Map;
 
 public class ResultView {
-    private static final String RESULT = "실행결과";
+    private static final String LADDER_RESULT = "사다리 결과";
+    private static final String GAME_RESULT = "실행결과";
     private static final String EMPTY = "";
     private static final String SINGLE_SPACE = " ";
     private static final String DOUBLE_SPACE = "  ";
     private static final String VERTICAL = "|";
     private static final String HORIZON = "-----";
     private static final String EMPTY_HORIZON = "     ";
+    private static final String PARTICIPANT_FORMAT = "%s%s%s";
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    private static final String PARTICIPANT_FORMAT = "%6s";
```